src/Microsoft.AspNetCore.Mvc.WebApiCompatShim/ApiController.cs

```diff
@@ -567,11 +567,10 @@ public void Validate<TEntity>(TEntity entity)
         /// </param>
         public void Validate<TEntity>(TEntity entity, string keyPrefix)
         {
-            var validatidationState = new ValidationStateDictionary();
+            var validationState = new ValidationStateDictionary();
```

Comment:
No, no. I'm positive we meant to introduce the new concept of `validatidation` states :trollface: